js/plugins/vertexai/src/gemini.ts

```diff
@@ -70,26 +70,55 @@ const SafetySettingsSchema = z.object({
 });
 
 const VertexRetrievalSchema = z.object({
-  datastore: z.object({
-    projectId: z.string().optional(),
-    location: z.string().optional(),
-    dataStoreId: z.string(),
-  }),
-  disableAttribution: z.boolean().optional(),
+  datastore: z
+    .object({
+      projectId: z
+        .string()
+        .describe('Project ID of the data store.')
+        .optional(),
+      location: z.string().describe('Location of the data store.').optional(),
+      dataStoreId: z
+        .string()
+        .describe(
+          'Data store resource name. Must provide either the full ' +
+            "data store resource name here as: 'projects/{project}/" +
+            'locations/{location}/collections/default_collection/' +
+            "dataStores/{data_store}' or provide {data_store} ID here " +
+            'and provide project ID, and location above.'
+        ),
+    })
+    .describe('Vertex AI Search data store details'),
+  disableAttribution: z
+    .boolean()
+    .describe(
+      'Disable using the result from this tool in detecting grounding ' +
```

Comment:
Something more specific we can say other than "tool"?

Reply:
Reworded it to 'search data'